attributes/default.rb

```diff
@@ -247,7 +247,7 @@
 default['datadog']['web_proxy']['port'] = nil
 default['datadog']['web_proxy']['user'] = nil
 default['datadog']['web_proxy']['password'] = nil
-default['datadog']['web_proxy']['skip_ssl_validation'] = nil # accepted values 'yes' or 'no'
+default['datadog']['web_proxy']['skip_ssl_validation'] = false
```

Comment:
Can we keep it nil here and just change the comment? This way it won't appear in the configuration if not set, thus using the default.

Reply:
Thanks for the feedback. I've addressed the comment.